src/sql/workbench/contrib/notebook/test/electron-browser/contentManagers.test.ts

```diff
@@ -131,4 +131,8 @@ suite('Local Content Manager', function (): void {
 		let displayOutput = <nb.IDisplayData>notebook.cells[0].outputs[0];
 		assert.equal(displayOutput.data['text/html'], '<div></div>');
 	});
+	test('Should create a new empty notebook if content is undefined', async function (): Promise<void> {
+		let content = await contentManager.loadFromContentString('');
```

Comment:
Could you add a case for undefined also?

Reply:
Sure!